modules/multi-architecture-modify-machine-set.adoc

```diff
@@ -4,26 +4,32 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="multi-architecture-modify-machine-set_{context}"]
+= Adding a multi-architecture compute machine set to your Azure cluster
 
-= Adding a multi-architecture compute machine set to your cluster
+After creating a multi-architecture cluster, you can add nodes with different architectures. 
 
-To add ARM64 compute nodes to your cluster, you must create an Azure compute machine set that uses the ARM64 boot image. To create your own custom compute machine set on Azure, see "Creating a compute machine set on Azure".
+You can add multi-architecture compute machines to a multi-architecture cluster in the following ways:
+
+* Adding 64-bit x86 compute machines to a cluster that uses 64-bit ARM control plane machines and already includes 64-bit ARM compute machines. In this case, 64-bit x86 is considered the secondary architecture.
+* Adding 64-bit ARM compute machines to a cluster that uses 64-bit x86 control plane machines and already includes 64-bit x86 compute machines. In this case, 64-bit ARM is considered the secondary architecture.
+
+To create a custom compute machine set on Azure, see "Creating a compute machine set on Azure".
 
 include::snippets/about-multiarch-tuning-operator.adoc[]
 
 .Prerequisites
 
 * You installed the OpenShift CLI (`oc`).
+* You created a 64-bit ARM or 64-bit x86 boot image.
+* You used the installation program to create a 64-bit ARM or 64-bit x86 single-architecture AWS cluster with the multi-architecture installer binary.
```

Comment:
just checking - this module seems to focus on Azure, should this prereq mention AWS?

Reply:
This is a great catch! I copied the pre-reqs but missed to update the "AWS" instances.